perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -104,5 +104,5 @@ then
 
   echo "Running e2e tests...."
   # $1 argument is refering to value of testInstalledPackage.
-  ./perfmetrics/scripts/run_e2e_tests.sh $RUN_E2E_TESTS_ON_INSTALLED_PACKAGE
+  time ./perfmetrics/scripts/run_e2e_tests.sh $RUN_E2E_TESTS_ON_INSTALLED_PACKAGE "true"
```

Comment:
Can we create a constant for SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE and add a comment similar to line 106